packages/xod-client/src/project/selectors.js

```diff
@@ -133,6 +133,26 @@ const addDeadFlag = R.curry(
   )(renderableNode)
 );
 
+// :: Patch -> RenderableNode -> RenderableNode
+const addVariadicErrors = R.curry(
+  (patch, renderableNode) => R.when(
+    R.compose(
+      XP.isVariadicPath,
+      R.prop('type')
+    ),
+    node => R.compose(
+      foldEither(
+        err => R.compose(
+          R.assoc('errors', [err]),
+          R.assoc('dead', true)
```

Comment:
What “dead” does here? I thought that a separate error class: dead references is one thing and variadic marker misuse is another.